tox.ini

```diff
@@ -16,6 +16,8 @@ deps =
     pyramid18: pyramid <= 1.8.99
     pyramid19: pyramid <= 1.9.99
     pyramid110: pyramid <= 1.10.99
+    sqlalchemy14: sqlalchemy <= 1.4
```

Comment:
from what I can tell this installed 1.4.0, it'd be better to set this at `<2.0` or `~=1.4`, but I think the former is better for the purposes here.

Reply:
Oops! Yes, that's correct.

Reply:
https://github.com/Pylons/pyramid_debugtoolbar/pull/388/commits/e3af7db7ede8aee495c4754999b0f568b50d18c7 fixed `sqlalchemy14` as suggested.